commands/utils/utils.go

```diff
@@ -136,3 +136,7 @@ func GetRelativeWd(fullPathWd, baseWd string) string {
 
 	return strings.TrimPrefix(fullPathWd, baseWd+string(os.PathSeparator))
 }
+
+func IsSimplifiedOutput(provider vcsutils.VcsProvider) bool {
```

Comment:
This function can be probably made private.

Reply:
done